ClubService.Application/Api/Exceptions/UserNotFoundException.cs

```diff
@@ -0,0 +1,15 @@
+namespace ClubService.Application.Api.Exceptions;
+
+public class UserNotFoundException : Exception
+{
+    public override string Message { get; }
+
+    public UserNotFoundException(Guid id, string username)
+    {
+        Message = $"{username} not found in tennis club with id: {id}";
```

Comment:
Maybe change message to `Message = $"User '{username}' not found in tennis club with id: {id}";`